README.md

```diff
@@ -107,6 +110,28 @@ const client = new Client({
 });
 ```
 
+#### DPoP authorization
+
+The client supports Oauth2+DPoP authorization ONLY FOR INTERNAL USE in service-to-service. Required methods can be configured during the initialization as 'authStrategy.' You must provide your implementation of the `getAccessToken` and `getDpopHeader` methods.
+
+Example:
+
+  ```ts
+  const client = new Client({
+    accountId: 'myAccountId',
+    authStrategy: {
+        getAccessToken: async (domainUrl: string) => {
+          // you implementation
+          return 'accessToken';
+        },
+        getDpopHeader: async (url: string, method: string, accessToken?: string) => {
```

Comment:
accessToken is optional as is not required to get the Dpop Header but can be included to add "extra security" , it is optional in the Internal DPOP client lib